src/ripple/shamap/SHAMapItem.h

```diff
@@ -33,87 +34,58 @@ namespace ripple {
 class SHAMapItem
 {
 private:
-    uint256 mTag;
-    Serializer mData;
+    uint256    tag_;
```

Comment:
Attempts to left justify code columns will be stymied by pervasive and ongoing search/replace rename refactorings